webpack.config.js

```diff
@@ -0,0 +1,188 @@
+const isProduction = process.env.NODE_ENV === 'production';
```

Comment:
here the issue was solved because webpack is no longer used as long we don't need no more the dev server 😉... checkout at [this branch](https://github.com/erikyo/wp-notify/tree/design_implementation) where I am integrating this style (with all your suggestions)